styles.css

```diff
@@ -0,0 +1,95 @@
+body {
+  width: 310px;
+}
+
+.nav-bars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+  width: 340px;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  width: 350px;
+  height: 470px;
+  padding-bottom: 50px;
+  padding-top: 50px;
```

Comment:
[ OPTIONAL ] I think you don't need padding-top here on the nav you just can remove it to match the Figma template.💪🏻